tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc

```diff
@@ -137,5 +141,35 @@ namespace EnergyPlus {
 
 	}
 
+	TEST_F( EnergyPlusFixture, HighTempRadiantSystemTest_SizeHighTempRadiantSystemScalableFlagSetTest )
+	{
+		int RadSysNum;
+		int SizingTypesNum;
+		
+		HighTempRadiantSystem::clear_state();
```

Comment:
@mjwitte And thanks for removing the extraneous clear_state.  Still not 100% on my understanding of those, as you can tell.